app/api/v1/endpoints/disease.py

```
@@ -0,0 +1,86 @@
+    disease_id: UUID,
+    db: AsyncSession = Depends(get_db)
+):
+    deleted_disease = await disease_crud.delete_disease(db, disease_id)
+    if deleted_disease is None:
+        raise HTTPException(status_code=404, detail="Disease not found")
+    return {"message": "Disease deleted successfully"}
+
+@router.get("/disease/{disease_id}", response_model=List[HospitalSchema])
+async def read_hospitals_by_disease(
+    disease_id: UUID,
+    db: AsyncSession = Depends(get_db)
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)